geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLTestUtils.scala

```diff
@@ -28,6 +28,7 @@ object SparkSQLTestUtils {
       .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
       .config("spark.kryo.registrator", classOf[GeoMesaSparkKryoRegistrator].getName)
       .config("spark.sql.crossJoin.enabled", "true")
+      .config("spark.ui.enabled", value = false)
```

Comment:
Is the default 'true'?

Reply:
yeah, this just prevents it from spinning up the spark UI jetty servlet for no reason

Reply:
the flag is mainly used by spark for unit testing i believe